docs/code-review.md

```diff
@@ -62,6 +63,10 @@ This approach can be summarized by the term **Minimum Viable Change (MVC)**. MVC
 
 If the developer decides to open a new issue from the current one or is not completely sure whether it is necessary to create a new issue or not, this must be commented in the original issue/PR.
 
+### PR Merge
+
+Once the PR is approved and the checks are passing, the author will be responsible for merging the PR or delegating the responsibility to another developer. There could be exceptions to this rule, for example, if the author is on vacation and the merge is "urgent", other developers can merge the PR to unblock the situation.
```

Comment:
Can we add what happen if the author is an external contributor? They are not allowed to merge the PR

Reply:
Sure! How do you think we should handle that situation?

I'm thinking that  
- If there are maintainers among the reviewer, they should merge the pr as they have better context
- If there are not maintainers among the reviewers, they would need to wait for a maintainer to take responsibility. They could ping the maintainers to remind them of the pr

Reply:
Maintainers are the only ones that can merge PRs. What I would do is:

- when all the requests from the maintainers have been completed(all the maintainers that have written something accepts the PR), merge
- I would ask for two approvals when an external contributor makes a PR, just to ensure that the PR is aligned with what we want